### PR TITLE
ci: basic composite gh action for ent

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: ci/action.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: entci
+description: continuous integration for ent projects
+inputs:
+  tidy:
+    description: run go tidy after generate
```
Comment (giautm) [inline ci/action.yaml:5]:
```suggestion
    description: run go mod tidy after generate
```

**Review by giautm — APPROVED** [review]
💯

File: ci/action.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: entci
+description: continuous integration for ent projects
+inputs:
+  tidy:
+    description: run go mod tidy after generate
```
Comment (a8m) [inline ci/action.yaml:5]:
```suggestion
    description: Run go mod tidy after generate
```

File: ci/action.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+name: entci
+description: continuous integration for ent projects
```
Comment (a8m) [inline ci/action.yaml:2]:
```suggestion
description: Continuous integration for Ent projects
```

File: ci/action.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+inputs:
+  tidy:
+    description: run go mod tidy after generate
+    required: false
+    default: ""
+  working-directory:
+    default: "."
+    required: false
+    description: where to run commands
```
Comment (a8m) [inline ci/action.yaml:11]:
```suggestion
    description: where to run commands
```

Any reason the descriptions do not start with uppercase?

**Review by a8m — APPROVED** [review]
💯

File: ci/action.yaml (hunk 4 above)
Comment (rotemtam) [reply ci/action.yaml:11]: Nope

**Pull request merged**